### DISCUSSION TITLE
Tell me your Favorite Football Player

### DISCUSSION THREAD

**Discussion by shiv-ko** [NAIST-SE/AttractingContributors#1042] [2024 Attraction Contributions]
Make contribute and tell me your favorite football player.
[Link](https://github.com/shiv-ko/AttractingContributors/blob/main/README.md)

**Comment by Samabena** [comment]
Hello @shiv-ko  , I recently made a contribution to your project , could you please contribute to my as well ? Following is the link to [my repository](https://github.com/Samabena/sighting) . Thanks in advance.

**Reply by shiv-ko** [reply]
Hello [@Samabena](https://github.com/Samabena).
Thank you for your contribution. I merged right now.
And I added India to your repository.

Thank you once again for your contributions.

**Comment by mustafizur-r** [comment]
Hello [@shiv-ko](https://github.com/shiv-ko) , I recently made a contribution to your project , I have added “Neymar" could you please contribute to my as well ? Following is the link to [my repository](https://github.com/MustafizurRU/AttractingContributors-My_Favorite_Sci-Fi_Books) . Thanks in advance.

**Reply by shiv-ko** [reply]
Hello [@MustafizurRU ]
Thank you for your contribution. I merged right now.
And I added to your repository as well.

Thank you once again for your contributions.

**Comment by thapakorn604** [comment]
Hi! I have added one player to your list -> https://github.com/shiv-ko/AttractingContributors/pull/11 please approve
Would you mind to contribute to my list ? -> https://github.com/NAIST-SE/AttractingContributors/discussions/1115

**Reply by shiv-ko** [reply]
Thank you for your contribution. I merged right now.
And I added to your repository as well.

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Reply by shiv-ko** [reply]
Thank you for your contribution. I merged right now.
And I added to your repository as well.
